src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -588,6 +588,14 @@ def test_process_declarations_intent():
     assert fake_parent.symbol_table.lookup("arg4").interface.access is \
         ArgumentInterface.Access.READWRITE
 
+    reader = FortranStringReader("integer, intent ( invalid ) :: arg5")
```

Comment:
I get the following pycodestyle error in code that you have not modified. Could you please fix.
555:1: E303 too many blank lines (3)

Reply:
Fixed it.